packages/tooling/fast-figma-plugin-msft/package.json

```diff
@@ -9,7 +9,7 @@
     "watch": "webpack --progress --watch --mode=production",
     "prettier": "prettier --config ../../../.prettierrc --write \"**/*.{ts,tsx}\"",
     "prettier:diff": "prettier --config ../../../.prettierrc \"**/*.{ts,tsx}\" --list-different",
-    "test": "yarn eslint && yarn build",
+    "test": "yarn eslint",
```

Comment:
What's the rational for *not* building as part of the test pass?

Reply:
Building after testing is redundant in more situations than not. I'll move these specific changes to a different PR with more details.

Reply:
#4336